src/drawing/DrawingPath.tsx

```diff
@@ -53,15 +56,17 @@ export const DrawingPath = ({ borderStrokeWidth = 0, isDecorated = false, points
                     <path
                         className="ba-DrawingPath-border"
                         d={pathCommands}
+                        fill="none"
                         stroke="#fff"
+                        strokeLinecap="round"
                         strokeWidth={borderStrokeWidth}
                         vectorEffect="non-scaling-stroke"
                     />
                 </g>
             )}
-            <path d={pathCommands} vectorEffect="non-scaling-stroke" />
+            <path ref={ref} d={pathCommands} strokeLinecap="round" vectorEffect="non-scaling-stroke" />
```

Comment:
It seems a little odd to assign the forwarded ref to an inner element instead of the root `<g>` tag. Should this be moved out to its own component? Should the other paths be moved to their own component(s)?